site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md

```diff
@@ -0,0 +1,54 @@
+
+We've heard a lot of interesting presentations and also met with [Yukihiro Matsumoto](https://github.com/matz), the creator of the Ruby language.
+
+![Matz and ManageIQ](https://i.imgur.com/BVYNubg.jpg)
+
+While we were at the conference, the others kept improving ManageIQ and merged more than a hundred PRs. As I wasn't there I couldn't follow all the changes, but now I have an opportunity to check them out while picking the most important ones for this post.
+
+## Improved
+
+While [Ladas](https://github.com/ladas) improved how we [save metrics](https://github.com/ManageIQ/manageiq/pull/15976) and [Tim](https://github.com/imtayadeway) added [event streams](https://github.com/ManageIQ/manageiq/pull/16021) as a new product feature we were exploring the Square of Heroes.
+
+![The Square of Heroes](https://i.imgur.com/UjJGoB9.jpg)
+
+## Fixed
+
+Back in the day, the *Service UI* was called *Self Service* and we missed some places were it should've been renamed. Thanks to [Allen](https://github.com/AllenBW) now it is [fixed and consistent](https://github.com/ManageIQ/manageiq-ui-service/pull/1030) across the repository. As we moved our GTL code to Angular, the policy simulation screen got broken and did not display quadicons but thanks to [Milan](https://github.com/mzazrivec) this is now [fixed](https://github.com/ManageIQ/manageiq-ui-classic/pull/2034). In the meantime we explored the square around the Matthias Church.
+
+![The Matthias Church](https://i.imgur.com/ATEQCPl.jpg)
+
+## New
+
+While we listened to a talk about failures and why it's bad that ruby is not slow enough, [Chris Hale](https://github.com/chalettu) added the ability to [disable custom buttons](https://github.com/ManageIQ/manageiq-ui-service/pull/1012) in the SUI and Moti added support for [publishing a VM](https://github.com/ManageIQ/manageiq/pull/15981) by oVirt.
+
+![Failures talk](https://i.imgur.com/7Bib9SV.jpg)
+
+## Deleted
+
+Keeping commented code that we don't plan to use is a perfect victim for deletion, thanks to [Brandon](https://github.com/bdunne) a lot of these comments are now [gone](https://github.com/ManageIQ/manageiq/pull/16033), some of them [were from 2010](https://github.com/ManageIQ/manageiq/pull/16032). Meanwhile we were being entertained by references to the Dunkirk movie in a talk about scalability.
+
+![Scalability talk](https://i.imgur.com/PUmU7I7.jpg)
+
+## Wrapping up
+
+The changes above are just a few from the many that happened around the project, feel free to check out our [GitHub organization](https://github.com/manageiq) page for more details.
+
+![Buda Castle](https://i.imgur.com/9nHHGji.jpg)
+
+Each year EuRuKo is organized in a different European city that is being chosen by the attendees of the previous conference. We are very happy that the next year it will be in Vienna that is just from an hour ride from our office in Brno. Here I would like to say thank you for the organizers and the speakers, it was an amazing event!
```

Comment:
Minor stuff: "just from an hour ride" should be "just an hour's ride" (or "just a one-hour ride") ; "thank you for" should be "thank you to"